src/react/databrowser/buckets/details/__tests__/Overview.test.jsx

```diff
@@ -0,0 +1,135 @@
+import * as T from '../../../../ui-elements/TableKeyValue2';
+import * as actions from '../../../../actions/s3bucket';
+import { Button, Toggle } from '@scality/core-ui';
+import { BUCKET_INFO_RESPONSE } from '../../../../actions/__tests__/utils/testUtil';
+import DeleteConfirmation from '../../../../ui-elements/DeleteConfirmation';
+import Overview from '../Overview';
+import React from 'react';
+import { bucketName } from '../../../../../js/mock/S3Client';
+import { reduxMount } from '../../../../utils/test';
+
+const BUCKET = {
+    CreationDate: 'Tue Oct 12 2020 18:38:56',
+    LocationConstraint: '',
+    Name: bucketName,
+};
+
+const TEST_STATE = {
+    uiBuckets: {
+        showDelete: '',
+    },
+    s3: {
+        bucketInfo: BUCKET_INFO_RESPONSE,
+    },
+    configuration: {
+        latest: {
+            locations: [{
+                'us-east-1': {
+                    isBuiltin: true,
+                    locationType: 'location-file-v1',
+                    name: 'us-east-1',
+                    objectId: '1060b13c-d805-11ea-a59c-a0999b105a5f',
+                },
+            }],
+        },
+    },
+    networkActivity: {
+        counter: 0,
+    },
+};
+
+describe('Overview', () => {
+    it('should render Overview component', () => {
+        const { component } = reduxMount(<Overview bucket={BUCKET}/>, TEST_STATE);
+
+        expect(component.find(Overview).isEmptyRender()).toBe(false);
+    });
+
+    it('should render Overview component with given infos', () => {
+        const { component } = reduxMount(<Overview bucket={BUCKET}/>, TEST_STATE);
+
+        const groupInfos = component.find(T.Group);
+        expect(groupInfos).toHaveLength(2);
+
+        // FIRST GROUP ITEMS TITLE
+        const firstGroupInfos = groupInfos.first();
+        expect(firstGroupInfos.find(T.GroupName).text()).toContain('General');
+
+        // FIRST GROUP ITEMS
+        const firstGroupInfosContentItems = firstGroupInfos.find(T.GroupContent).find(T.Row);
+        let firstContentItem = firstGroupInfosContentItems.first();
```

Comment:
use `const` here. 
`firstContentItem` is being reassigned with unrelated value.
You can call it `firstItemFirstGroup`.

Reply:
done